Samples/GitHubLabeler/GitHubLabeler/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Configuration;
+using System.Threading.Tasks;
+
+namespace GitHubLabeler
+{
+    internal static class Program
+    {
+        private static async Task Main(string[] args)
+        {
+            if (args.Length != 1)
```

Comment:
Is it possible to simplify this sample a bit, so that people don't have to learn some syntax of a simple command line program? As far as I see the sample should just be a straight train of a model, followed by a label, right? Or are we showing how the artifact is preserved across trainings?

Reply:
Yes, the intention was to demonstrate how it is used in real life where you train it just once and inference many times. Completely agree that for the tutorial samples we want them to be simple train-evaluate-predict. This one is an end-to-end app that you would use for labeling issues. I will probably move it to separate folder for "apps infused with ML" examples.